sdk/storage/Azure.Storage.Files.DataLake/src/DataLakePathClient.cs

```diff
@@ -2112,6 +2114,477 @@ internal virtual void SetNameFieldsIfNull()
         }
         #endregion Set Access Control
 
+        #region Set Access Control Recursive
+        /// <summary>
+        /// The <see cref="SetAccessControlRecursive"/> operation sets the
+        /// Access Control on a path and subpaths
+        ///
+        /// For more information, see <see href="https://docs.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/path/update" />.
+        /// </summary>
+        /// <param name="accessControlList">
+        /// The POSIX access control list for the file or directory.
+        /// </param>
+        /// <param name="progressHandler">
+        /// <see cref="Progress{AccessControlChanges}"/> callback where caller can track progress of the operation
+        /// as well as collect paths that failed to change Access Control.
+        /// </param>
+        /// <param name="options">
+        /// Optional <see cref="AccessControlChangeOptions"/> with additional controls.
+        /// </param>
+        /// <param name="continuationToken">
+        /// Optional continuation token that can be used to resume previously stopped operation.
+        /// </param>
+        /// <param name="cancellationToken">
+        /// Optional <see cref="CancellationToken"/> to propagate
+        /// notifications that the operation should be cancelled.
+        /// </param>
+        /// <returns>
+        /// A <see cref="Response{AccessControlChangeResult}"/> that contains summary stats of the operation.
+        /// </returns>
+        /// <remarks>
+        /// A <see cref="RequestFailedException"/> will be thrown if
+        /// a failure occurs.
+        /// </remarks>
+        public virtual Response<AccessControlChangeResult> SetAccessControlRecursive(
+            IList<PathAccessControlItem> accessControlList,
+            IProgress<Response<AccessControlChanges>> progressHandler,
+            AccessControlChangeOptions options = default,
+            string continuationToken = default,
```

Comment:
@kasobol-msft @amnguye, I think the idea is continuation is a required parameter on subsequent calls.

Reply:
in sunny day scenario there's no interaction with continuationToken, and single api call would follow it till end of data set.
continuation token would be provided in case of:
- manual iteration - together with maxbatches (which causes APi to stop at some point if specified)
- error/failure recovery
can go to options imo.

Reply:
Let's at least change the order to put it above the `options` so it shows up sooner in intellisense then?

Reply:
fixed.

Reply:
fixed.